app/views/boards/show.html.erb

```diff
@@ -1,15 +1,23 @@
 <body>
  <div class="container">
  <table class="block-center">
-     <% 4.times do %>
+     <% 4.times do |row| %>
       <tr class="even">
- 	     <% 8.times do %>
- 	      <td></td>	 
+ 	     <% 8.times do |even_cell| %>
+       <% if Piece.where(file: even_cell + 1, rank: row * 2 + 1)[0]%>
```

Comment:
`GamesController#show` is already pulling up the right game based on `params[:id]`, so you can just use the Active Record association between games and pieces to get the proper set of pieces.